docs/sources/flow/reference/components/otelcol.receiver.vcenter.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.receiver.vcenter/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.receiver.vcenter/
+title: otelcol.receiver.vcenter
+description: Learn about otelcol.receiver.vcenter
+---
+
+# otelcol.receiver.vcenter
+
+`otelcol.receiver.vcenter` accepts metrics from a 
+vCenter or ESXi host running VMware vSphere APIs and
+forwards it to other `otelcol.*` components.
+
+> **NOTE**: `otelcol.receiver.vcenter` is a wrapper over the upstream
+> OpenTelemetry Collector `vcenter` receiver from the `otelcol-contrib`
+> distribution. Bug reports or feature requests will be redirected to the
+> upstream repository, if necessary.
+
+Multiple `otelcol.receiver.vcenter` components can be specified by giving them
+different labels.
+
+Make sure that you fulfill the [prerequisites](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/vcenterreceiver/README.md#prerequisites) before using this component.
+
+The full list of metrics that can be collected can be found [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/vcenterreceiver/documentation.md)
```

Comment:
I had a look at this page, and a few things are not clear to me:
* There are [optional metrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/vcenterreceiver/documentation.md#optional-metrics). How are these enabled in the Agent? There is a yaml config in the Collector docs, but I don't see how this translates to Agent config.
* Are the [resource attributes](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/vcenterreceiver/documentation.md#resource-attributes) enabled in the Agent by default as well? Can they be disabled?